packages/collector/test/tracing/messaging/nats-streaming/test.js

```diff
@@ -12,229 +12,252 @@ const { v4: uuid } = require('uuid');
 const constants = require('@instana/core').tracing.constants;
 const supportedVersion = require('@instana/core').tracing.supportedVersion;
 const config = require('../../../../../core/test/config');
-const delay = require('../../../../../core/test/test_util/delay');
-const { expectAtLeastOneMatching, retry } = require('../../../../../core/test/test_util');
+const { delay, stringifyItems, expectAtLeastOneMatching, retry } = require('../../../../../core/test/test_util');
 const ProcessControls = require('../../../test_util/ProcessControls');
 const globalAgent = require('../../../globalAgent');
 
 const agentControls = globalAgent.instance;
-
 const mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
 
 mochaSuiteFn('tracing/nats-streaming', function () {
-  this.timeout(config.getTestTimeout() * 2);
+  describe('tracing is enabled', function () {
```

Comment:
FYI: The diff is big because `tracing is disabled` was not executed. I grouped it into the main `describe`.